### DISCUSSION TITLE
Python 自學影片

### DISCUSSION THREAD

**Discussion by danwan1027** [1101-GS4719-C/Python-Community-GS4719-C#695] [General]
學習是永無止盡的，想學好一門程式語言僅僅夠過學校的課程是永永不夠，想精通一門程式語言是需要花費大量的時間和精力尋找資源，透過自學的方式不斷精進自己。以下是 freeCodeCamp 在 youtube 上推出的 Python 教學影片，若是想繼續精進或是擔心跟不上進度的同學都很適合觀看此影片。
影片連結：https://www.youtube.com/watch?v=rfscVS0vtbw

**Comment by Yunhans** [comment]
怎麼點進去是rickroll

**Reply by danwan1027** [reply]
沒可能吧😢

**Comment by a45320614** [comment]
我想在網路上學程式時也都是看freeCodeCamp.org!
內容相當淺顯易懂
重點是教學真的非常詳細
影片常常都是10個小時起跳的!

**Comment by williamyen0623** [comment]
感謝分享，受益良多！
